easybuild/tools/github.py

```diff
@@ -1487,3 +1521,12 @@ def find_easybuild_easyconfig(github_user=None):
 
     eb_file = os.path.join(eb_parent_path, fn)
     return eb_file
+
+def fetch_pr_data(pr, pr_target_account, pr_target_repo, github_user):
```

Comment:
@migueldiascosta Should we add a dedicated test for `fetch_pr_data` in `test/framework/github.py`?

Reply:
@boegel will do